content/en/docs/instrumentation/java/_index.md

```diff
@@ -74,8 +74,8 @@ using our BOM to keep the versions of the various components in sync.
 
 ```kotlin
 dependencies {
-  implementation(platform("io.opentelemetry:opentelemetry-bom:1.10.1"))
-  implementation("io.opentelemetry:opentelemetry-api")
+  implementation(platform("io.opentelemetry:opentelemetry-bom:1.12.0"))
+  implementation("io.opentelemetry:opentelemetry-api:1.12.0")
```

Comment:
Note the update here. Is this ok @open-telemetry/java-approvers?

Reply:
Since we have the bom we don't need a version here, and I think it's idiomatic to leave it out as previous. Otherwise it gets harder to reason about what's controlling what version (e.g., if the version of api was 1.13.0 I'm not really sure if it would actually be 1.13.0 or the bom would take precedence)

Reply:
> Since we have the bom we don't need a version here, and I think it's idiomatic to leave it out as previous.

Thanks for the feedback @anuraaga. Done: I reverted the addition to the api dep.